README.md

```diff
@@ -140,7 +144,29 @@ clj -M:membrane.term screenshot --play play-msgcat.sh --height 14 \
 ```
 The screenshot command produces `terminal.png`:
 <!-- generated by script/regen-screenshots.sh -->
-![play-deep-diff-scheme-example](doc/images/screenshot-msgcat-scheme.png)
+![play-msgcat-scheme-example](doc/images/screenshot-msgcat-scheme.png)
+
+## Fonts
+
+You can specify a font family and size.
+The font must be OS installed and is assumed to be monospace.
```

Comment:
It should be possible to load a font by providing a path the the font file, but there are some issues in membrane that I should probably fix before trying to support it in `membrane.term`. I've created a related membrane issue at https://github.com/phronmophobic/membrane/issues/27.